docs/podman-image-trust.1.md

```diff
@@ -14,18 +14,27 @@ podman\-trust - Manage container registry image trust policy
 REGISTRY[/REPOSITORY]
 
 # DESCRIPTION
-Manages which registries you trust as a source of container images based on its location.  The location is determined by the transport and the registry host of the image.  Using this container image `docker://docker.io/library/busybox` as an example, `docker` is the transport and `docker.io` is the registry host.
+Manages which registries you trust as a source of container images  based on its location.  The location is determined
+by the transport and the registry host of the image.  Using this container image `docker://docker.io/library/busybox`
+as an example, `docker` is the transport and `docker.io` is the registry host.
 
-The trust policy describes a registry scope (registry and/or repository).  This trust can use public keys for signed images.
+Trust is defined in **/etc/containers/policy.json** and is enforced when a user attempts to pull
+an image from a registry that is managed by policy.json.  The trust policy describes a registry scope (registry and/or repository).  This trust can use public keys for signed images.
 
-Trust is defined in **/etc/containers/policy.json** and is enforced when a user attempts to pull an image from a registry that is managed by policy.json.
+The scope of the trust is evaluated from most specific to the least specific. In other words, a policy may be defined for an entire registry.  Or it could be defined for  for a particular repository in that registry. Or it could be defined down to a specific signed image inside of the registry.
```

Comment:
`s/for  for/for/`

Reply:
fixed, and I swear fixed for the third time.  Thanks for spotting that.